libs/@local/hash-isomorphic-utils/src/blocks.ts

```diff
@@ -203,13 +203,19 @@ export const fetchBlock = async (
   return await promise;
 };
 
-export const paragraphBlockComponentId =
-  "https://blockprotocol.org/blocks/@hash/paragraph";
+/**
+ * @todo-0.3 replace this temporary domain with blockprotocol.org
+ * https://app.asana.com/0/1203358502199087/1203788113163116/f
+ */
+export const blockProtocolHubOrigin =
+  "https://blockprotocol-git-03.stage.hash.ai";
```

Comment:
Can this be made an env var with a default value? It might make it easier to test things in the future

Reply:
Yes, good idea – I've done this in https://github.com/hashintel/hash/pull/1920/commits/a6b33347e0400a5aa6bb2ca2db94466b16c20a3f

I've kept the default as the temporary domain, as I think it will be less work for me to come back and replace this than to ensure that everyone has the temporary domain in an environment variable on their machine (with those who miss the prompt encountering bugs / starting Slack threads)